scala/BUILD

```diff
@@ -9,6 +9,9 @@ toolchain_type(
 scala_toolchain(
       name = 'default_toolchain_impl',
       scalacopts = [],
+      library = "//external:io_bazel_rules_scala/dependency/scala/scala_library",
+      compiler = "//external:io_bazel_rules_scala/dependency/scala/scala_compiler",
```

Comment:
Why are you using //external: dependencies instead of @io_bazel_rules_scala//dependency/scala/scala_compiler?